templates/web/base/waste/bulky/intro.html

```diff
@@ -2,6 +2,27 @@ <h3>Before you start your booking</h3>
 <ol>
     <li>Requesting a bulky waste collection usually takes approximately <strong>10 minutes</strong></li>
     <li>You can request up to <strong>[% c.cobrand.bulky_items_maximum | numwords %] items per collection</strong></li>
+    [% IF cobrand.moniker == 'kingston' || cobrand.moniker == 'sutton' %]
+    <li>The price you pay depends how many items you would like collected:
+        <ul class="bulky-intro-pricing">
+            [% IF cobrand.moniker == 'sutton' %]
+            <li class="bulky-intro-pricing">
+                Up to 4 items = £35.05
```

Comment:
Can we read all these numbers from the database configuration so we don't have to update them? :)
(That would also fix that it's 3 items 35.05, 6 58.73)